youtube_dl/extractor/bandcamp.py

```diff
@@ -47,6 +47,15 @@ def _real_extract(self, url):
         mobj = re.match(self._VALID_URL, url)
         title = mobj.group('title')
         webpage = self._download_webpage(url, title)
+        album = self._search_regex(
+            r'(?ms).*?title\s*?:\s*?"(?P<album>.*?)",',
+            webpage, 'album', fatal=False)
+        artist = self._search_regex(
+            r'(?ms)var EmbedData = .*?[{,]\s*artist:\s*?"(?P<artist>.*?)",$',
```

Comment:
I wanted to use the artist that was included in the EmbedData var (which I can now see is the first instance of artist, usually). In most cases, three of four instances of the artist are the same, with one being null (in TralbumData).  Should I just remove the artist field altogether?